packages/typography/src/Link.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useMemo } from 'react';
+import ArrowRightIcon from '@leafygreen-ui/icon/dist/ArrowRight';
+import Box, { ExtendableBox } from '@leafygreen-ui/box';
+import { useBaseFontSize } from '@leafygreen-ui/leafygreen-provider';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { HTMLElementProps, createDataProp } from '@leafygreen-ui/lib';
+import { typeScale1, typeScale2 } from './styles';
+import { __TARGET__ } from '.';
+
+const anchorDataProp = createDataProp('anchor-container');
+
+const linkStyles = css`
+  text-decoration: none;
+  color: ${uiColors.blue.base};
+  cursor: pointer;
+
+  &:focus {
+    outline: none;
+  }
+`;
+
+const underline = css`
+  background-repeat: repeat-x;
+  background-position: 0 1em;
+  background-size: 3px 3px;
+
+  ${anchorDataProp.selector}:hover & {
+    background-image: linear-gradient(
+      to right,
+      ${uiColors.gray.light2} 100%,
+      ${uiColors.gray.light2} 0
+    );
+  }
+
+  ${anchorDataProp.selector}:focus & {
+    background-image: linear-gradient(to right, #9dd0e7 100%, #9dd0e7 0);
+  }
+`;
+
+const arrowRightIconPersist = css`
+  transform: translate3d(3px, 0, 0);
+`;
+
+const arrowRightIconHover = css`
+  height: 10px;
+  width: 10px;
+  opacity: 0;
+  transform: translate3d(-3px, 0, 0);
+  transition: all 100ms ease-in;
+
+  ${anchorDataProp.selector}:hover & {
+    opacity: 1;
+    transform: translate3d(3px, 0, 0);
+  }
+`;
+
+const openInNewTabStyles = css`
+  margin-bottom: -1px;
+  margin-left: -1px;
+  margin-right: -2px;
+`;
+
+const OpenInNewTab = (props: JSX.IntrinsicElements['svg']) => {
+  return (
```

Comment:
Why not simply add this to the Icon library? (cc @audroue we should probably do that in the Sketch library as well)

Reply:
What should we call the icon?

Reply:
I can add to the library